app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,15 @@
+<footer class="footer">
+  <small>
+    <%= t(".description_title", href: link_to(t("ruby_on_rails_tut"), Settings.links.rails_tutorial.home_page)).html_safe %>
```

Comment:
html_safe để làm gì e ?

Reply:
ban đầu em truyền đoạn html <a></a> thuần vào i18n thì sẽ gặp các lỗi liên quan đến csp; truyền các đoạn mã html chưa đáng tin cậy, nên bị escape thành text hết ạ. Nên thêm html_safe cho nhận biết là đoạn mã đó an toàn và có thể được thực thi ạ.